### PR TITLE
Add gwctl

### PR DESCRIPTION
**Pull request by MorrisLaw**
Add a CLI to go-workers2.

### CODE REVIEW

File: cmd/retries.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+	goworkersctl stats --a 127.0.0.1 --p 8080`,
+	RunE: runGetRetries,
+}
+
+func init() {
+	rootCmd.AddCommand(retriesCmd)
+}
+
+func runGetRetries(cmd *cobra.Command, args []string) error {
```
Comment (stefannegrea) [inline cmd/retries.go]: Might be better for consistency to rename this to `runRetries` to match the command name.

File: cmd/stats.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+	goworkersctl stats --a 127.0.0.1 --p 8080`,
+	RunE: runGetStats,
+}
+
+func init() {
+	rootCmd.AddCommand(statsCmd)
+}
+
+func runGetStats(cmd *cobra.Command, args []string) error {
```
Comment (stefannegrea) [inline cmd/stats.go]: Same as the other run method, what do you think about renaming this to `runStats`.

**Comment by stefannegrea** [comment]
Could you please make the binary file is not included in the commit? Also, maybe add it to the ignore list?

File: cmd/retries.go (hunk 1 above)
Comment (MorrisLaw) [reply cmd/retries.go]: agreed, will update

File: cmd/stats.go (hunk 2 above)
Comment (MorrisLaw) [reply cmd/stats.go]: ditto

**Comment by stefannegrea** [comment]
This PR is ready for review.

File: cmd/root.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.PersistentFlags().StringVar(&hostAddress, "a", "localhost", "Host address for a specific goworkers2 instance.")
+	rootCmd.PersistentFlags().StringVar(&port, "p", "8080", "Port number for a specific goworkers2 instance.")
+	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
```
Comment (lecda) [inline cmd/root.go:42]: is the toggle flag needed?  what does it do?  this looks like a placeholder.

Comment (stefannegrea) [reply cmd/root.go:42]: ohh, I think this can be deleted ...

Comment (stefannegrea) [reply cmd/root.go]: Indeed it was not needed, removed the line.

**Review by lecda — APPROVED** [review]
👍

**Pull request merged**